tests/dependency_tests.cpp

```diff
@@ -18,13 +18,6 @@ TEST_CASE("Testing dependency validating locations") {
    CHECK(d.location() == "larryk85/does-not-exist");
 
    CHECK(!d.is_valid_location());
-
-   // TODO: reinstate these tests when support for general git repos and archives is added
-   //d.location("https://github.com/larryk85/cturtle");
```

Comment:
why did you remove it? It sounds like the time to uncomment it.

Reply:
Original idea lost its sense, because in location we are storing short / relative version i.e. `org/project`.
Only tests that would be valid in this file are already done above and they are testing short version.

The tests that are testing full github URL are added to location_tests.cpp and net_utils_tests.cpp